### ISSUE TITLE
If you have one chunk named 'noiseperlin' and another named 'noiseperlinmodel' and you try to include it using $noiseperlinmodel, it will instead include the first chunk and leave behind the 'model' part

### ISSUE THREAD

**Issue by epitaque** [Makio64/shader-loader#4]
Example filestructure:

```
shaders
    chunks
        noiseperlin.glsl
        noiseperlinmodel.glsl
    terrain.vert
```

Inside terrain.vert:

```
$noiseperlinmodel
```

It actually inserts noiseperlin.glsl and leaves the word 'model' in the generated terrain.vert

**Comment by Makio64** [comment]
Thanks for your report, I'll fix this today👍

**Comment by Makio64** [comment]
fixed in 1.3.0, sorry for the wait
